lesson10/index.js

```diff
@@ -0,0 +1,67 @@
+const ul = document.getElementById('js-target');
+
+function addLoading() {
+  const img = document.createElement('img');
+  img.id = 'loading-img';
+  img.src = 'loading-circle.gif';
+  img.alt = 'loading';
+  ul.appendChild(img);
+}
+
+function removeLoading() {
+  const loadingImg = document.getElementById('loading-img');
+  loadingImg.remove();
+}
```

Comment:
so nits

I feel it is easy to understand without making it variable.

```suggestion
function removeLoading() {
  document.getElementById('loading-img').remove();
}
```

Reply:
I agree with haru-san!

Reply:
Thanks for the review.
I see, it makes it easier to read.
[710c07e](https://github.com/tom106gunma/JavaScript-assignment/pull/11/commits/710c07eb671065abf077fbd03f1019511544d106)